runner/src/server/plugins/engine/pageControllers/RepeatingSummaryPageController.ts

```diff
@@ -118,10 +118,8 @@ export class RepeatingSummaryPageController extends PageController {
   }
 
   buildRows(state, response) {
-    let form_session_identifier = "";
-    if (response.request.query.form_session_identifier) {
-      form_session_identifier = `&form_session_identifier=${response.request.query.form_session_identifier}`;
-    }
+    let form_session_identifier =
+      response.request.query.form_session_identifier ?? "";
```

Comment:
Due to the new update to multinput components the form_session_identifier is being built in the buildTextFieldRows method, so this was cause a double up issue when adding the query params